app/models/miq_report/seeding.rb

```diff
@@ -68,14 +68,15 @@ def sync_from_file(filename, dir, typ)
       rpt[:template_type] = typ
       rec = self.find_by_filename(rpt[:filename])
 
+      type_titleized = typ.titleize
       if rec
-        if rec.filename && (rec.file_mtime.nil? || rec.file_mtime.utc < rpt[:file_mtime])
-          _log.info("#{typ.titleize}: [#{rec.name}] file has been updated on disk, synchronizing with model")
+        if rec.filename && (rec.file_mtime.nil? || rec.file_mtime.utc.to_i < rpt[:file_mtime].to_i)
```

Comment:
> Correct, because using the Time object is more precise (carrying also the fraction). It's not wrong, it just warns you that it may be more accurate than what you expect (down to nanoseconds).

indeed, but see this comment with the comparison of the time as extracted from file versus the one stored in the db, they always have some diff and hence the files are always reloaded.
https://github.com/ManageIQ/manageiq/pull/3475#issue-95400804